docs/src/1_Mechanics/05_Vulnerabilities.yaml

```diff
@@ -187,3 +167,98 @@ Wanted:
     Type: Vulny
     XP: [-1,-2]
 
+Clumsy, Minor:
```

Comment:
This implementation is fine for now, but I think we could find a way for this to be implemented more gracefully in the future, selecting from a list of attributes and major/minor